libs/novui/src/json-schema-components/shared.tsx

```diff
@@ -26,8 +26,13 @@ const toggleButtonRecipe = cva({
   base: {
     gap: 'margins.icons.Icon20-txt',
 
-    '&:not(:disabled)': { cursor: 'pointer' },
-    '&:hover:not(:disabled)': { opacity: 'hover' },
+    cursor: 'pointer',
+    _disabled: {
+      cursor: 'default',
+    },
+    '&:hover:not(:disabled)': {
+      opacity: 'hover',
+    },
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** Minor UX tweak following new Step Controls merge‏